project/Dependencies.scala

```diff
@@ -0,0 +1,12 @@
+import sbt._
+
+object Dependencies {
+
+  val FinatraVersion = "2.10.0"
+  val LogbackVersion = "1.2.2"
+
+  val Test = Seq(
+    "org.scalatest" %% "scalatest" % "3.0.3" % "test,it",
+    "org.scalacheck" %% "scalacheck" % "1.13.5" % "test,it"
```

Comment:
As implementing fakes is so easy in scala, we'll probably won't need mock unless we need the spy capabilities of the mock. However, once we need it, we can add it.